### ISSUE TITLE
Improve and automate social media presence

### ISSUE THREAD

**Issue by simlu** [simlu/voxelshop#168]
Not sure yet what best to do here, however we want more presence in social because:

More exposure means more ppl using and more ppl using is good because better chance of more contributers which means more features! :)

**Comment by funabab** [comment]
These type of things take time... I think there are few things to consider here. 

1. Don't be mute about what's going on: keeping mute about development may result to people thinking the software is dead. 

2. Give people good reason to use your app: I think in this aspect voxelshop performs well but well may not be enough. Apart from the fact that voxelshop is open source (which is a great move), to a designer this doesn't matters much. Most of the stuff voxelshop supports,  others support them to some extent. Maybe when the animation/other features are finally ready, we might see some new faces. 

There are a lot more but i dont just want this post to get long. And i will see what i can contribute (ex java squad). Need to wait a little more for some features to be released be4 i can start using voxelshop properly.

**Comment by simlu** [comment]
@funabab Thanks for the comment. Great stuff.

The problem is really that VoxelShop tries to satisfy both, programmers/developers and designers/artists. Unfortunately I'm not an artist, so a lot of features that I pushed are very useful for developers (The mesh optimization algorithm for 3D models is still unmatched by any other voxel editor including Qubicle but few people know about it). Have you tried the option to texture voxels? No other editor has that.

The goal is really to teach people about what the program can do already (there is a lot that people don't know about), and to keep them up to date with development.

At this point "throwing more features at it" is really hard to justify, since there are already so many features here. 

"Most of the stuff voxelshop supports, others support them to some extent." - somewhat true. If you are looking for free editors with layer support VoxelShop is your only option. I'm not saying VoxelShop is perfect, but I think the return on promoting it, getting feedback and iterating is way higher than spending another half year developing big features... I've been there and trying to change it.

Hope that makes sense.

**Comment by funabab** [comment]
Yeah makes sense but am not against promoting it (would even love to help) but to be sincere there are still somethings basic things missing currently in voxelshop like line, fill, extrude etc that other voxel engine supports and also it would be great if voxelshop support some form of animation (cause it was created with animation feature in mind) and if you also take a closer look into the forum you will see it one of the most requested feature. Am just making my $0.2, BTW i read it somewhere that the other popular free voxel editor will support layers in next version.

**Comment by simlu** [comment]
@funabab And those things will come. A lot of tools are planned for 1.9.0 (the release coming after the this one). And animation support will also come.

It's really just a matter of prioritization. I really wanna show off some of the cools things you can already do with VoxelShop and not be like "yeah but we really need animation nothing else matters". It's a matter of staying motivated...

**Comment by simlu** [comment]
Back on topic: Would you be interested in helping managing social media accounts and helping with creating content? I really want to push some short, exciting videos highlighting some features out...

**Comment by funabab** [comment]
Wow great cant wait for 1.9.0

**Comment by funabab** [comment]
It depends on what you mean by managing account. Do you mean creating contents with voxelshop and posting it on Twitter with #voxelshop. If it's that then i already planned that when the extrude/fill/primitives tool is available (still practicing art and getting my heads around voxelshop). And also that's what i think magica has that voxelshop is really missing. But i may also use magica from time to time due to it renderer. 

2. I may not be able to help with videos cause i have a bad voice :D. 

3 if i have time am thinking of creating a small open-source game using voxelshop and godotengine if that will also help.

**Comment by simlu** [comment]
Im thinking to use hootsuite to manage, but what social accounts do we want?

Reddit, Twitter, Tumblr, Pinterest, Snapchat(?), Facebook, Instagram... What else? What are people using? Twitter seems dead compared to two years ago...

**Comment by simlu** [comment]
https://www.quora.com/What-are-the-differences-between-the-way-you-use-Quora-Twitter-Facebook-Pinterest-Reddit-Tumblr-Instagram-and-the-other-social-media-sites-you-use

**Comment by simlu** [comment]
Which also ties in with https://github.com/simlu/voxelshop/issues/14

**Comment by kevzettler** [comment]
Don't write off Twitter. I get all my indie voxel game news there. There is an active indie game community. MagicaVoxel is very active on twitter. they have a hashtag: https://twitter.com/hashtag/MagicaVoxel?src=hash

There's also a popular #voxeldailies 
https://twitter.com/hashtag/voxeldailies?src=hash

In which voxel artists post daily works. We need to tap that network and get people trying VoxelShop

**Comment by funabab** [comment]
Simlu Twitter is dead?  What will you say of Facebook then :D

**Comment by simlu** [comment]
Thanks for the comment @kevzettler

This might be an argument to ramp up the rendering in VoxelShop. Nicer screenshots = more shares.

My next idea is to do really short videos (~30-60 sec) and explain smaller parts and then promote these to all platforms.

**Comment by kevzettler** [comment]
> This might be an argument to ramp up the rendering in VoxelShop. Nicer screenshots = more shares.

Absolutely that's why artists have gravitated to MagicaVoxel. It gives you that polished ambient occlusion shading for free by default.

**Comment by simlu** [comment]
I'm thinking we might use something better supporting expensive rendering: https://github.com/simlu/voxelshop/issues/73

Maybe that will be next after tooling?

**Comment by kevzettler** [comment]
I feel that skeletal animation would be the feature that has more potential for viral social sharing. Any Voxel animation post I've seen blows up. Currently, voxel animation posts are limited to artists who have committed to Quibicle or imported to Unity. 

I feel if VoxelShop lowered the barrier to entry for voxel animation that could be its niche

**Comment by simlu** [comment]
Agreed. We are currently tied to JPCT and would be even more so once animation is implemented. So maybe time to change the rendering before rather than after...

**Comment by funabab** [comment]
Simlu, kevzettler is right. I have also noticed that a lot of folks on Twitter are obsessed with voxel animation and as i said earlier once this feature is implemented will definitely see some new faces. BTW kevzettler are you on Twitter? and did you mind if we can hang around (and discuss and make voxelshop better), i go by my name on Twitter.

**Comment by funabab** [comment]
Also to anyone else, i also wish to know all of you. Maybe we should hang around on Twitter.

**Comment by simlu** [comment]
Sounds good. Will consider prioritizing animation. However let's see what the benefit return on some other promotions is first...

**Comment by MrTopom** [comment]
Twitter: @acteedrop

**Comment by simlu** [comment]
Twitter: @whiteflux

**Comment by rubengc** [comment]
Twitter: @rubengece

I will try to be more active helping us!

**Comment by Bremaweb** [comment]
You mentioned automation. You could look into zapier.com for automating some social media posting. You can setup triggers so when something is committed on github it creates a tweet / facebook post about it. That would be of more interest to developers but it's an easy way to get content out on your social platforms. There is also buffer.com which lets you queue up posts for social media so you can sit down for 30 minutes and create the posts you want to send out for the next few days. Both of these have free plans.

**Comment by simlu** [comment]
@Bremaweb I actually like Hootsuite for queuing social as well. I guess I have to spend some time and investigate...

**Comment by thatguyminib** [comment]
I think when you get the animation for VoxelShop up and running doing a nice feature montage video would really help out! The animation niche is also the way to go I feel! I am super excited for it lol
